gobblin-core/src/main/java/gobblin/metrics/GobblinMetrics.java

```diff
@@ -228,6 +234,19 @@ public Timer getTimer(String prefix, String... suffixes) {
   /**
    * Start metric reporting.
    *
+   * @param configuration configuration properties
+   */
+  public void startMetricReporting(Configuration configuration) {
```

Comment:
It was used at some point when I was working on these changes, but not anymore. I figure I'd leave it because it might be useful in the future, although I don't mind removing it.